test/e2e/performanceprofile/functests/10_performance_ppc/ppc.go

```diff
@@ -79,7 +80,7 @@ var _ = Describe("[rfe_id: 38968] PerformanceProfile setup helper and platform a
 			cmdArgs := []string{
 				fmt.Sprintf("%s:%s:z", mustgatherDir, mustgatherDir),
 				ntoImage,
-				"--mcp-name=worker",
+				fmt.Sprintf("--mcp-name=%s", mcpName),
```

Comment:
by default (in case the env variable isn't set) the target mcp would be worker-cnf, see here:
https://github.com/openshift/cluster-node-tuning-operator/blob/master/test/e2e/performanceprofile/functests/utils/consts.go#L38
does that make any difference in the expected results? is the gathered data have both mcps?

Reply:
so when running under SNO environment we want to set ROLE_WORKER_CNF=master and run the tests

Reply:
the change then solves SNO perfectly; what about MNO , do we always have worker-cnf mcp in the env from which we gathered the must-gather?

Reply:
so in our CI for MNO , our CI creates a must-gather before running PPC tests, and by default we have worker-cnf label set.